addons/sale/views/sale_portal_templates.xml

```diff
@@ -136,10 +136,10 @@
             <div class="row o_portal_sale_sidebar">
                 <!-- Sidebar -->
                 <t t-call="portal.portal_record_sidebar" id="sale_order_portal_sidebar">
-                    <t t-set="classes" t-value="'d-print-none col-lg-3 col-xl-4'"/>
+                    <t t-set="classes" t-value="'d-print-none col-lg-4 col-xl-3'"/>
 
                     <t t-set="title">
-                        <h2 t-field="sale_order.amount_total" data-id="total_amount"/>
+                        <h2 t-field="sale_order.amount_total" data-id="total_amount" class="text-break"/>
```

Comment:
This is actually not recommended but as we fix the width of our columns, the text would either overflow or wrap. Not sure if another approach can be applied there